path_login_test.go

```diff
@@ -64,6 +64,7 @@ func setupBackend(t *testing.T, cfg testConfig) (closeableBackend, logical.Stora
 			data = map[string]interface{}{
 				"bound_issuer":           "https://team-vault.auth0.com/",
 				"jwt_validation_pubkeys": ecdsaPubKey,
+				"jwt_supported_algs":     []string{string(jose.ES256)},
```

Comment:
Should we be testing with `cap/jwt` (i.e. `jwt.ES256`) values instead?

Reply:
Ya, can use the values from `cap/jwt` here. I've done that in 22e3e09. There was a package name collision with `square/go-jose.v2/jwt` though, so I changed its identifier. Lmk your thoughts!